autopatch/autopatch.go

```diff
@@ -181,8 +181,10 @@ func PatchResource(api huma.API, path *huma.PathItem) {
 			return
 		}
 
+		ressourcePath := findRelativeRessourcePath(ctx.URL().Path, put.Path)
```

Comment:
Minor typo:

```suggestion
		resourcePath := findRelativeResourcePath(ctx.URL().Path, put.Path)
```

Also has to be fixed further down below.